thermo/EquationsOfState.ipynb

```diff
@@ -0,0 +1,414 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Plot the result\n",
+    "\n",
+    "# Internal energy\n",
+    "plot(T, p, uIdeal, uRK, uCoolProp, \"Relative Internal Energy [kJ/kg]\")\n",
+    "# Enthalpy\n",
+    "plot(T, p, hIdeal, hRK, hCoolProp, \"Relative Enthalpy [kJ/kg]\")\n",
+    "# Entropy\n",
+    "plot(T, p, sIdeal, sRK, sCoolProp, \"Relative Entropy [kJ/K]\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The thermodynamic properties such as internal energy, enthalpy, and entropy are plotted against the operating pressure at a constant temperature T = 300 K. Three equations follow each other closely at low pressures (P > 10 Bar). However, the ideal gas EoS departs significantly from the observed behavior of gases near the critical regime (P$_{\\rm {crit}}$ = 73.77 Bar).\n",
+    "\n",
+    "The ideal gas EoS does not consider inter-molecular interactions and the volume occupied by individual gas particles. At low temperatures and high pressures, inter-molecular forces become particularly significant due to a reduction in inter-molecular distances. Additionally, at high pressures, the volume of individual molecules becomes significant. Both of these factors contribute to the deviation from ideal behavior at high pressures. The cubic Redlich-Kwong EoS, on the other hand, predicts thermodynamic properties accurately near the critical regime. "
```

Comment:
```suggestion
    "The ideal gas EoS does not consider inter-molecular interactions and the volume occupied by individual gas particles. At low temperatures and high pressures, inter-molecular forces become particularly significant due to a reduction in inter-molecular distances. Additionally, at high density, the volume of individual molecules becomes significant. Both of these factors contribute to the deviation from ideal behavior at high pressures. The cubic Redlich-Kwong EoS, on the other hand, predicts thermodynamic properties accurately near the critical regime. "
```

Reply:
I believe this correction is more accurate, but I'm not totally sure.

Reply:
I think an increase in the density is an effect (and not the cause) of reduced volume. At high pressures, the gas molecules are crowded and occupy larger proportion of the total volume, which in turn increase the density.

Reply:
Corrected.